app/models/actor.rb

```diff
@@ -66,11 +64,9 @@ class Actor < ApplicationRecord
               with: /\A\d{15,15}[0-9X]{1,1}\z/,
               message: 'must be valid',
               allow_nil: true
-            }
-
-  validates :orcid,
+            },
             presence: true,
-            on: :from_user
+            unless: -> { psu_id.present? }
```

Comment:
I'm not sure this `unless psu_id.present?` is entirely in the right place. As I'm reading this, the intended behavior is, "orcid is required if psu_id is missing". This accomplishes that goal, but it _also_ affects the format regex too. I believe what will happen with this change is that, if psu_id is present, then I can type anything I want into the orcid field and its format will be accepted.

Reply:
Yep. That makes sense to me.

Reply:
@rschenk I've updated this and added a test case to reflect it.